labrad/protocol.py

```diff
@@ -298,24 +298,24 @@ def call_handler():
                                   msgCtx, data, listener)
                             traceback.print_exc()
                     d = call_handler()
-                    if not async:
+                    if not _async:
                         yield d
 
     # message handling
-    def addListener(self, listener, source=None, context=None, ID=None, async=True, args=(), kw={}):
+    def addListener(self, listener, source=None, context=None, ID=None, _async=True, args=(), kw={}):
```

Comment:
This looks like a breaking change.

Reply:
Yes it is, as mentioned in the initial description. I've also posted a possible workaround I could integrate. There is no way around renaming this variable.